test/e2e/e2ethanos/services.go

```diff
@@ -491,3 +491,39 @@ func NewMemcached(name string) *e2e.ConcreteService {
 
 	return memcached
 }
+
+func NewToolsBucketWeb(sharedDir string, name string, bucketConfig client.BucketConfig, routePrefix, externalPrefix string) (*Service, error) {
```

Comment:
We are not using the `sharedDir` argument anywhere. Let's remove it.

Reply:
addressed and rebase done :+1: